pkg/vsphere/disk/vmdk.go

```diff
@@ -62,9 +62,18 @@ func (v *Vmdk) Mount(op trace.Operation, uri *url.URL, persistent bool) (string,
 }
 
 func LockedVMDKFilter(vm *mo.VirtualMachine) bool {
+	if vm == nil {
```

Comment:
Why? I would argue calling this function with nil is a programming error and we should panic :)